modal/_tunnel.py

```diff
@@ -17,16 +17,47 @@
 
 @dataclass
 class Tunnel:
+    """A port forwarded from within a running Modal container. Created by `modal.forward()`.
+
+    This is an EXPERIMENTAL API and may change in the future.
+    """
+
     host: str
 
     @property
     def url(self) -> str:
+        """Get the public HTTPS URL of the forwarded port."""
         return f"https://{self.host}"
 
 
 @contextlib.asynccontextmanager
 async def _forward(port: int) -> AsyncIterator[Tunnel]:
-    """Forward a port from within a running Modal container."""
+    """Expose a port publicly from inside a running Modal container, with TLS.
```

Comment:
Btw it would be good if `_forward` can take an `Optional[_Client]` as an argument (and default to `from_env` if it's `None`) – that way it should be pretty easy to write a test for it, since you can inject a reference to the test service client. Not super important since the RPC interface is fairly trivial right now.